uint8 IDLE = 0
uint8 GOTO = 1
uint8 SHOT = 2

nav_msgs/Odometry desired_odometry
uint8 type
